### PR TITLE
🌱 Add .github directory from CAPI

### LINKED ISSUE #121: 🌱 Add .github directory from CAPI
We should bring over https://github.com/kubernetes-sigs/cluster-api/tree/master/.github/ISSUE_TEMPLATE and https://github.com/kubernetes-sigs/cluster-api/blob/master/.github/PULL_REQUEST_TEMPLATE.md so we can push for tagged PRs to make release notes easier to generate.

### PR DESCRIPTION
**Pull request by fenglixa**
**What this PR does / why we need it**:
Add .github directory for cluster-api-provider-nested
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #121

/cc @christopherhein @gyliu513

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<!-- please add a icon to the title of this PR (see https://sigs.k8s.io/cluster-api-provider-nested/VERSIONING.md), and delete this line and similar ones -->
```
Comment (christopherhein) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: Any chance we can link out to CAPI's copy instead and not have to copy over the `VERSIONING.md` file?
```suggestion
<!-- please add a icon to the title of this PR (see https://sigs.k8s.io/cluster-api/VERSIONING.md), and delete this line and similar ones -->
```

**Review by christopherhein — COMMENTED** [review]
Looks great to me, only one comment about linking to `VERSIONING.md` in CAPI and removing that additional file.

wdyt?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (fenglixa) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: OK, @christopherhein , let's keep the link of cluster-api here

**Comment by fenglixa** [comment]
Thanks @christopherhein , for your review. I addressed your comments. Please take a look again.

**Comment by christopherhein** [comment]
/lgtm
/approve

Thank you for doing this, we appreciate the help!

**Pull request merged**